text/0000-compiler-team-rolling-leads.md

```diff
@@ -0,0 +1,282 @@
+compiler team member.
+
+# Reference-level explanation
+[reference-level-explanation]: #reference-level-explanation
+
+The Rust Compiler Team uses a rolling co-leadership model of governance, with a
+senior lead and a junior lead each serving the team as team representatives and
+decision owners, for a total of two to five years.
+
+## Team Representative
+
+The leads represent the compiler team by speaking with Rust users, individual
+contributors, or organizations seeking to support development of Rust; they also
+represent the team by engaging with other teams in the Rust project.
+
+A team lead must have a rough understanding (at least) of what tasks fall under
+the remit of the compiler team, and which are better suited for another one of
+the Rust teams.
+
+A team lead should be aware of what large scale initiatives are happening within
+the compiler, so that they can speak in an informed manner about what issues in
+the compiler are being addressed, and which issues are not likely to be
+addressed in the near term.
+
+A team lead should be aware of what painpoints (technical, social, etc) that the
+compiler team is suffering from most. The leads should be prepared to provide
+advice on how others can provide support, and the leads should be prepared to
+reach out on behalf of the team to identify external stakeholders to the team
+who could drive progress forward on resolving issues that face the team.
+
+The leads' representation of the team should manifest itself via structured
+communication, such as blog posts on https://blog.rust-lang.org/inside-rust/
+
+## Decision Owner
+
+The leads own decisions on behalf of the compiler team.
+
+Most choices made by the team are consensus-driven by the usual "FCP all-but-two
+with no concerns" process.
+
+The leads own making decisions about urgent issues or ones with a specific
+deadline. For example, deciding what to do about a critical release-blocking bug
+should happen before the release, preferably long before. Likewise,
+beta-backport decisions need to be made in time for the backport to happen
+before the beta is lifted to stable.
+
+Finally, when adverse events happen, the leads are responsible for reviewing
+what decisions or processes led to the event, and taking action to prevent
+future occurrences of the same event. Examples of this include the incr-comp bug
+that plagued the 1.52.0 release, which led to a 1.52.1 release four days later
+(and three steering meetings as follow-up).
+
+## Time Commitment, Expectations, and Competencies
+
+Let us recall that any compiler team member is allowed to:
+
+* drive progress on backlogged work,
+* draft steering meeting proposals, and often write the associated steering meeting document to drive the meetings,
+* solicit individuals to form working groups to address important problems,
+* take on the resolution of unassigned or abandoned P-critical or P-high issues, and
+* drive larger initiatives related to the language.
```

Comment:
I agree that compiler work should be explicitly indicated here, since there are plenty of things that are entirely compiler-internal that we would want spelled out as large initiative.

But the whole point of this section, in many ways, is to spell out things that someone might *think* is part of the role of the T-compiler leader, and yet in fact **isn't** -- its rather the role of any T-compiler member. (Arguably its the role of any project contributor who wants to stand up and do the work!)

Reply:
Changed my mind, went with Wesley's original suggestion, see comment on commit here for more details: https://github.com/rust-lang/rfcs/pull/3262/commits/e4f3c57fced4553e9e7c9acc4da873d5ef3b3b9a